src/__tests__/Credentials-test.js

```diff
@@ -304,12 +310,24 @@ describe('createPersonalSignRequest()', () => {
 
 describe('createVerification()', () => {
   beforeAll(() => mockresolver())
-  it('has correct payload in JWT for an attestation', async () => {
-    return uport.createVerification({sub: 'did:uport:223ab45', claim: {email: 'bingbangbung@email.com'}, exp: 1485321133 + 1}).then(async (jwt) => {
-      const decoded = await verifyJWT(jwt)
-      return expect(decoded).toMatchSnapshot()
+  describe('use explicit expiry time', () => {
+    it('has correct payload in JWT for an attestation', async () => {
+      return uport.createVerification({sub: 'did:uport:223ab45', claim: {email: 'bingbangbung@email.com'}, exp: 1485321133 + 1}).then(async (jwt) => {
+        const decoded = await verifyJWT(jwt)
+        return expect(decoded).toMatchSnapshot()
+      })
     })
   })
+
+  describe('use expiresIn', () => {
```

Comment:
Add a test case that expects an error to be thrown when both `exp` and `expiresIn` are passed?